identity/app/views/consentJourney.scala.html

```diff
@@ -151,7 +151,6 @@ <h3 class="manage-account__switch-title">&nbsp;</h3>
 @displayJourney = {
     <div class="
         identity-consent-journey
-        @{if (!user.statusFields.hasRepermissioned.getOrElse(false)){"identity-consent-journey--with-alert"}}
```

Comment:
adding and removing this class doesn't make a difference
I can't find references to 'with-alert' in the project